### PR TITLE
Add 0.7 release note

### PR DESCRIPTION
**Pull request by qiujian16**
Signed-off-by: Jian Qiu <jqiu@redhat.com>

### CODE REVIEW

File: content/en/community/releases.md [hunk 1]
```diff
@@ -3,9 +3,31 @@ title: Releases
 weight: -20
 ---
 
-Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.5.0`. Continue reading to view upcoming releases:
+Open Cluster Management has approximately a three to four month release cycle. The current release is `v0.7.0`. Continue reading to view upcoming releases:
 
-## `v0.7.0`, on 25th, March 2022
+## `v0.7.0`, on 1st, April 2022
+
+The Open Cluster Management team is exicted to announce the release of OCM v0.7.0! We mainly focused on enhancing user experience in this release by introducing a bunch of new commands in `clusteradm`. Notable changes including:
```
Comment (mprahl) [inline content/en/community/releases.md:10]: Could you please mention the introduction of the `PolicySet` API?

File: content/en/community/releases.md [hunk 2]
```diff
@@ -3,9 +3,31 @@ title: Releases
...
+
+See details in the release changelogs::
+- registration v0.7.0 [changelog](https://github.com/open-cluster-management-io/registration/blob/v0.7.0/CHANGELOG/CHANGELOG-v0.7.md)
+- work v0.7.0 [changelog](https://github.com/open-cluster-management-io/work/blob/v0.7.0/CHANGELOG/CHANGELOG-v0.7.md)
+- placement v0.4.0 [changelog](https://github.com/open-cluster-management-io/placement/blob/v0.4.0/CHANGELOG/CHANGELOG-v0.4.md)
+- addon-framework v0.3.0 [changelog](https://github.com/open-cluster-management-io/addon-framework/blob/v0.3.0/CHANGELOG/CHANGELOG-v0.3.md)
+- registration-operator v0.7.0 [changelog](https://github.com/open-cluster-management-io/registration-operator/blob/v0.7.0/CHANGELOG/CHANGELOG-v0.7.md)
+- cluster-proxy v0.2.0 [repo](https://github.com/open-cluster-management-io/cluster-proxy)
+- managed-serviceaccount v0.2.0 [repo](https://github.com/open-cluster-management-io/managed-serviceaccount)
+- clusteradm v0.2.0 [changelog](https://github.com/open-cluster-management-io/clusteradm/releases/tag/v0.2.0)
```
Comment (mprahl) [inline content/en/community/releases.md:27]:
```suggestion
- clusteradm v0.2.0 [changelog](https://github.com/open-cluster-management-io/clusteradm/releases/tag/v0.2.0)
- governance policy propagator v0.7.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-propagator/blob/main/CHANGELOG/CHANGELOG-v0.7.0.md)
- config policy controller v0.7.0 [changelog](https://github.com/open-cluster-management-io/config-policy-controller/blob/main/CHANGELOG/CHANGELOG-v0.7.0.md)
- policy spec sync controller v0.7.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-spec-sync/blob/main/CHANGELOG/CHANGELOG-v0.7.0.md)
- policy template sync controller v0.7.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-template-sync/blob/main/CHANGELOG/CHANGELOG-v0.7.0.md)
- policy status sync controller v0.7.0 [changelog](https://github.com/open-cluster-management-io/governance-policy-status-sync/blob/main/CHANGELOG/CHANGELOG-v0.7.0.md)
```

File: content/en/community/releases.md (hunk 1 above)
Comment (qiujian16) [reply content/en/community/releases.md:10]: sure, do you have a brief description on PolicySet?

Comment (dhaiducek) [reply content/en/community/releases.md:10]:
```
The new `policyset` API provides a way to logically group `policy` objects from a namespace, share placement, and report on overall status for the set.
```

**Review by yue9944882 — COMMENTED** [review]
LGTM

**Pull request merged**